Theme.html

```diff
@@ -5,6 +5,7 @@
 		<link rel="stylesheet" href="https://static.tumblr.com/2wliono/oG8ps6bg4/jsondrop.css">
 		<link rel="stylesheet" href="https://static.tumblr.com/2wliono/PXMps6cfk/colour.css">
 		<script src="https://static.tumblr.com/zctqgvy/TaXps6sqb/jsondrop.js"></script>
+		<meta name="text:Auto-Expand Depth" content="2"/>
```

Comment:
i was thinking of a `select:` type option for foolproofing, but this does give more freedom...

Reply:
if you want foolproofing don't use a json theme

Reply:
fair